fmriprep/config.py

```diff
@@ -624,3 +630,40 @@ def init_spaces(checkpoint=True):
 
     # Make the SpatialReferences object available
     workflow.spaces = spaces
+
+
+def load_previous_config(output_dir, work_dir, participant=None):
+    """
+    Search for existing config file, and carry over previous options.
+    If a participant label is specified, the output directory is searched for
+    the most recent config file. If no config is found, the working directory is searched.
+
+    Returns
+    -------
+    success : bool
+        Existing config found and loaded
+
+    """
+    conf = None
+    if participant:
+        # output directory
+        logdir = Path(output_dir) / 'fmriprep' / f'sub-{participant[0]}' / 'log'
+        for f in logdir.glob(f'**/{CONFIG_FILENAME}'):
+            conf = conf or f
+            if f.stat().st_mtime > conf.stat().st_mtime:
+                conf = f
+
+    if conf is None:
+        # work directory
+        conf = Path(work_dir) / f'.{CONFIG_FILENAME}'
+        if not conf.exists():
+            conf = None
+
+    # load existing config
+    if conf is not None:
+        print(f"Reusing config file {conf}")
+        load(conf)
+        execution.run_uuid = _gen_run_uuid()
```

Comment:
```suggestion
        load(conf, skip=("execution.run_uuid",))
```

Maybe it would be more generalizable to inform the load function of fields that should be skipped? (this suggestion requires the implementation of the skip arg)

That said, skipping ``execution.run_uuid`` will be necessary/desired in all instances?